types/angular-bootstrap-lightbox/index.d.ts

```diff
@@ -40,7 +40,11 @@ declare namespace angular.bootstrap.lightbox {
     }
 
     export interface ILightbox {
-        openModal(images:ILightboxImageInfo[], index:number):void;
+        openModal(
+            images:ILightboxImageInfo[],
+            index:number,
+            modalParams?: any, // angular.ui.bootstrap.IModalSettings
```

Comment:
IMO, this is just an object:
https://github.com/compact/angular-bootstrap-lightbox/blob/master/src/lightbox-service.js#L267
so maybe:
```suggestion
            modalParams?: { [key: string]: any }, // angular.ui.bootstrap.IModalSettings
```

Reply:
Thanks.